Formula/bit.rb

```diff
@@ -1,22 +1,23 @@
 require "language/node"
+
 class Bit < Formula
   desc "Distributed Code Component Manager"
   homepage "https://www.bitsrc.io"
-  url "https://bitsrc.jfrog.io/bitsrc/bit-brew/stable/bit/0.10.3/bit-0.10.3-brew.tar.gz"
-  sha256 "e843e29b34ba8c93c68503f288be28c841c159b67a54bcc88063328167a32785"
+  url "https://registry.npmjs.org/bit-bin/-/bit-bin-0.10.7.tgz"
+  sha256 "d033b76974be7103933abe1fc937297d29c19aa1cdabbba8d5032ebd9d4f72f0"
+  head "https://github.com/teambit/bit"
 
   bottle do
     cellar :any_skip_relocation
+    sha256 "afc34fb0172b2eb1a579b4334a084c0df77b7ad6977a11d3641a72dca6211487" => :high_sierra
```

Comment:
this shouldn't be modified manually

Reply:
@ilovezfs - my mistake thanks for the fast reply, i fixed this